### ISSUE TITLE
Show number of registered push notification users

### ISSUE THREAD

**Issue by timoludwig** [Integreat/cms-django#589]
### Motivation

Region managers who send push notifications should be able to see the number of registered push notification users.
This is determined by the number of subscribers to a specific channel.

### Proposed Solution

Find out how  to retrieve this information from the Firebase API.

**Comment by dkehne** [comment]
Any updates here @timobrembeck ?

**Comment by timobrembeck** [comment]
> Any updates here @timobrembeck ?

No, nobody is assigned to this issue, thus nobody is working on it.

**Comment by svenseeberg** [comment]
https://gist.github.com/timrijckaert/218be90541a7285e21455fafa2962dea

**Comment by JoeyStk** [comment]
We understand that this would be really helpful, but after some research from our side it came out that this is not easy to do and now we're wondering how much effort should be spend on this. Also we cannot promise that it is possible at all.
@hauf-toni could you please find out how important this issue is and give us a rough estimate on how many hours we should spend on this to decide whether we should continue work on this. Thank you in advance :)

**Comment by hauf-toni** [comment]
It’s a nice-to-have rather than a must-have. If the number of push notification subscribers would directly impacts regional managers decision-making, then it might be worth exploring. if it’s only for informational purposes and doesn’t change how notifications are sent, the high implementation cost does not justify it. @laravoelk what do you think?
Based on my insights right now would recommend closing this as not planned.
